src/java/org/apache/cassandra/utils/btree/BTree.java

```diff
@@ -1823,7 +1824,7 @@ private static <V> int find(Object[] btree, V from, Comparator<V> comparator)
 
     private static boolean isStopSentinel(long v)
     {
-        return v == Long.MAX_VALUE;
+        return v == STOP_SENTINEL_VALUE;
```

Comment:
Could you update docstring on `BTree#accumulate`? Also looks like we stopped using the sentinel Long.MIN_VALUE here a while ago, seems like a mistake in the comment.

Not something that should be addressed in this patch, but feels like we should have a separate AbortableLongAccumulator that signals early termination out of band to make this even safer.